### PR TITLE
add gatsby-starter-directus

### PR DESCRIPTION
**Pull request by muhzulzidan**
A starter template to build websites using  Directus as CMS & Gatsby for FrontEnd

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -6959,3 +6959,21 @@
     - Uses SCSS for easy-to-understand naming
     - Google Analytics support
     - Responsive design
+- url: https://gatsby-directus-starter.netlify.app/
+  repo: https://github.com/muhzulzidan/gatsby-starter-directus
+  description: A starter template to build websites using  Directus as CMS & Gatsby for FrontEnd 
+  tags:
```
Comment (aaaakshat) [inline docs/starters.yml:6965]: Refer to [categories.yml](https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml) to see the valid tags that can be added here. You cannot have tags like `Styling:Sass` (Use instead: `Styling:SCSS`), `CMS:Directus` etc. as they aren't part of `categories.yml` if you believe they are missing you can request additions to the file. However, we recommend using the available categories.

Comment (muhzulzidan) [reply docs/starters.yml:6965]: so i can't do pull request using github UI ?

Comment (muhzulzidan) [reply docs/starters.yml:6965]: i made the changes here : https://github.com/gatsbyjs/gatsby/pull/26457

and I think it is still not accepted by circleci...

**Comment by aaaakshat** [comment]
This PR can be closed as it has been updated in #26457

**Pull request closed without merging**